### ISSUE TITLE
Title Editor needs more work: unable to change Title size,style and effect (underline)

### ISSUE THREAD

**Issue by YoussefAMansour** [OpenShot/openshot-qt#3868]
Title tool not working properly
When I add title to my project. I click on "Change Font button", I select my desired options, I get the preview of it in the "Sample" window but when I click OK but, it remained unchanged.

Usually the style changes, But size and effects don't.
When I again click on "Change Font button", the selected options goes back to default.

System Details

Windows 10
OpenShot Version 2.5.1
To Reproduce

Go to 'Title'
Click on 'Title'
Select 'camera border' (as I am using it now).
Click on "Change Font" and do some changes to the text (Size and effect).
See error
Expected behavior
I wanted the title tool to change the title's characteristics (Style,Size & Effects) according to my wish. And have selected options selected even when I got back to edit the same titles.

Additional context
Basically the whole title tool has to be remastered, to save some the user/editor from headaches.

Overall the application is AWESOME, but please fix these issues.

Thank you.

**Comment by ferdnyc** [comment]
Partly fixed in development via #3727, partly duplicate of #3847, #3462, #3676.
